patchwork-level-generators/src/main/resources/patchwork-level-generators.accesswidener

```diff
@@ -0,0 +1,2 @@
+accessWidener v1 named
+accessible method net/minecraft/world/level/LevelGeneratorType <init> (ILjava/lang/String;)V
```

Comment:
Nitpick: all the files I've seen end in a newline, maybe this should too?

Otherwise LGTM